### PR TITLE
Dont follow to symlink.

### PR DESCRIPTION
**Pull request by ArseniyK**

### CODE REVIEW

**Comment by MeanEYE** [comment]
Is there a need to copy links at all?

File: application/operation.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ class Option:
 	SILENT = 5
 	SILENT_MERGE = 6
 	SILENT_OVERWRITE = 7
+	FOLLOW_TO_SYMLINK = 8
```
Comment (MeanEYE) [inline application/operation.py]: Rename to `FOLLOW_SYMLINK` as symlink is on source, following _to_ implies destination. :)

**Comment by ArseniyK** [comment]
UPDATED

File: application/gui/main_window.py [hunk 2]
```diff
@@ -2316,7 +2316,8 @@ def load_config(self):
 					'trash_files': True,
 					'reserve_size': False,
 					'automount_start': False,
-					'automount_insert': False
+					'automount_insert': False,
+					'follow_to_symlink': False
```
Comment (MeanEYE) [inline application/gui/main_window.py]: Can you rename this to `follow_symlink`?

**Comment by ArseniyK** [comment]
UPDATED

**Pull request merged**